docs/contributors/advanced/migration-issues.md

```diff
@@ -0,0 +1,874 @@
+# Proposed migration
```

Comment:
This page is [already added](https://github.com/ubuntu/ubuntu-project-docs/edit/main/docs/how-ubuntu-is-made/processes/proposed-migration.md) from the UMH, so you won't need to include it in this PR :)